sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Rules/PropertyGetterWithThrowBase.cs

```diff
@@ -67,6 +67,13 @@ protected sealed override void Initialize(SonarAnalysisContext context)
                         c =>
                         {
                             var throwExpression = GetThrowExpression(c.Node);
+
+                            // This is the case in rethrow - throw;

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS125)